frontend_vue/src/App.vue

```diff
@@ -1,12 +1,13 @@
 <template>
+  <ModalsContainer />
```

Comment:
what is this? does this mean all modals pop through this thing?

Reply:
yes it's component from the 3rd party library we use for modals here. https://vue-final-modal.org/
It's renderless, so basically like a <template> - as per my understanding.
It doesn't mount as a regular component, it's just there to host a modal if there's any